src/main/java/co/cask/gcp/bigquery/BigQuerySource.java

```diff
@@ -249,18 +247,9 @@ public void onRunFinish(boolean succeeded, BatchSourceContext context) {
   @Path("getSchema")
   public Schema getSchema(Request request) throws Exception {
     try {
-      BigQuery bigquery;
-      BigQueryOptions.Builder bigqueryBuilder = BigQueryOptions.newBuilder();
-      if (request.serviceFilePath != null) {
-        bigqueryBuilder.setCredentials(loadServiceAccountCredentials(request.serviceFilePath));
-      }
+      BigQueryOptions.Builder bigqueryBuilder = GCPUtils.getBigQuery(request.project, request.serviceFilePath);
+      BigQuery bigquery = bigqueryBuilder.build().getService();
       String project = request.project == null ? ServiceOptions.getDefaultProjectId() : request.project;
-      if (project == null) {
```

Comment:
Nothing here should change if you're using the latest code on release/0.11. If this is supposed to go to develop, please make sure the release changes get merged back to develop then rebase.